index.html

```diff
@@ -63,157 +68,196 @@
+            </p>
+            <ul>
+              <li class="lang">html</li>
+              <li class="lang">css</li>
+              <li class="lang">javaScript</li>
+            </ul>
+            <a class="moreabout" href="#">See Project</a>
+          </div>
         </div>
 
         <!-- 2nd work! -->
         <div class="works">
-          <img src="images/works/work2.svg" alt="" />
-          <h2>Tonic</h2>
-          <ul>
-            <li class="worksubtitle">CANOPY</li>
-            <li class="bullets">Back End Dev</li>
-            <li class="bullets">2015</li>
-          </ul>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul>
-            <li class="lang">html</li>
-            <li class="lang">css</li>
-            <li class="lang">javaScript</li>
-          </ul>
-          <a class="moreabout" href="#">See Project</a>
+            <img src="images/works/work2.svg" alt="" />
+          <div class="works-desk">
+            <h2>Tonic</h2>
+            <ul>
+              <li class="worksubtitle">CANOPY</li>
+              <li class="bullets">Back End Dev</li>
+              <li class="bullets">2015</li>
+            </ul>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul>
+              <li class="lang">html</li>
+              <li class="lang">css</li>
+              <li class="lang">javaScript</li>
+            </ul>
+            <a class="moreabout" href="#">See Project</a>
+          </div>
         </div>
 
         <!-- 3rd work! -->
         <div class="works">
           <img src="images/works/work3.svg" alt="" />
-          <h2>Tonic</h2>
-          <ul>
-            <li class="worksubtitle">CANOPY</li>
-            <li class="bullets">Back End Dev</li>
-            <li class="bullets">2015</li>
-          </ul>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul>
-            <li class="lang">html</li>
-            <li class="lang">css</li>
-            <li class="lang">javaScript</li>
-          </ul>
-          <a class="moreabout" href="#">See Project</a>
+          <div class="works-desk">
+            <h2>Tonic</h2>
+            <ul>
+              <li class="worksubtitle">CANOPY</li>
+              <li class="bullets">Back End Dev</li>
+              <li class="bullets">2015</li>
+            </ul>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul>
+              <li class="lang">html</li>
+              <li class="lang">css</li>
+              <li class="lang">javaScript</li>
+            </ul>
+            <a class="moreabout" href="#">See Project</a>
+          </div>
```

Comment:
Kindly adjust the fonts and spacing so that they look exactly like the Figma template as required. 👍 

Your Project
![image](https://user-images.githubusercontent.com/66112760/161398699-5ddc55ab-3f73-4bb1-9a67-901d3a8b8369.png)

Figma Template
![image](https://user-images.githubusercontent.com/66112760/161398719-21eddca8-fa19-4680-b8a9-384cd7d131ab.png)